src/metabase/api/card.clj

```diff
@@ -1193,13 +1193,15 @@ saved later when it is ready."
           stats             (upload/load-from-csv! driver db-id schema+table-name csv-file)
           ;; Syncs are needed immediately to create the Table and its Fields; the scan is settings-dependent and can be async
           table             (sync-tables/create-or-reactivate-table! database {:name table-name :schema (not-empty schema-name)})
+          table-id          (u/the-id table)
+          _set_is_upload    (t2/update! Table table-id {:is_upload true})
```

Comment:
To avoid a second DB call, `is_upload` should be set when the table is created by `sync-tables/create-or-reactivate-table!`

Reply:
Feels like a strange extension of `create-or-reactivate-table!` to pass in arbitrary values to update the table with. I don't think a single DB call saved is worth complexifying the code that way

Reply:
Yup, fair